api/simqueue/oauth.py

```diff
@@ -69,12 +69,15 @@ async def can_view(self, collab_id):
         # if that fails, check if it's a public collab
         try:
             collab_info = await get_collab_info(collab_id, self.token["access_token"])
-        except ValueError:
-            return False
+        except:
```

Comment:
I think we should avoid raising `HTTPException` in `oauth.py`, all user interaction should be kept in `queue.py` or `statistics.py`